ecc/blocks/video-content-component/video-content-component.css

```diff
@@ -0,0 +1,3 @@
+.video-content .field-label {
```

Comment:
This is probably fine but we technically should always wrap everything inside .block-name